packages/datagateway-table/src/pageHead.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I think this file is a leftover from a previous merge conflict or something - the component was renamed to PageContainer and this component is completely unused & untested. Please delete it.